src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/processors/probation/ProbationUnmergeEventProcessor.kt

```diff
@@ -0,0 +1,37 @@
+package uk.gov.justice.digital.hmpps.personrecord.message.processors.probation
+
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.personrecord.client.model.sqs.domainevent.DomainEvent
+import uk.gov.justice.digital.hmpps.personrecord.jpa.repository.PersonRepository
+import uk.gov.justice.digital.hmpps.personrecord.model.types.SourceSystemType.DELIUS
+import uk.gov.justice.digital.hmpps.personrecord.service.EventKeys
+import uk.gov.justice.digital.hmpps.personrecord.service.TelemetryService
+import uk.gov.justice.digital.hmpps.personrecord.service.type.TelemetryEventType.UNMERGE_MESSAGE_RECEIVED
+
+@Component
+class ProbationUnmergeEventProcessor(
+  val telemetryService: TelemetryService,
+  val personRepository: PersonRepository,
+) : BaseProbationEventProcessor() {
+
+  fun processEvent(domainEvent: DomainEvent) {
+    telemetryService.trackEvent(
+      UNMERGE_MESSAGE_RECEIVED,
+      mapOf(
+        EventKeys.SOURCE_CRN to domainEvent.additionalInformation?.sourceCrn,
+        EventKeys.TARGET_CRN to domainEvent.additionalInformation?.targetCrn,
+        EventKeys.EVENT_TYPE to domainEvent.eventType,
+        EventKeys.SOURCE_SYSTEM to DELIUS.name,
+      ),
+    )
+    getProbationCase(domainEvent.additionalInformation?.targetCrn!!).fold(
+      onSuccess = {
+        log.info("Successfully mapped unmerged record")
+      },
+      onFailure = {
+        log.error("Error retrieving offender detail: ${it.message}")
```

Comment:
This is a situation where we cannot find a record for the person we are trying to unmerge? We should probably make that clearer in the error message - the above is a bit generic